package.json

```diff
@@ -476,5 +477,11 @@
         }
       }
     ]
+  },
+  "__metadata": {
+    "id": "4f1304da-7e65-48f1-9126-0a143e2e5ef2",
+    "publisherDisplayName": "Apache Software Foundation",
+    "publisherId": "4dbc1d1a-d64b-46f8-8756-1c234855f645",
```

Comment:
Where do `id` and `publisherId` come from? Do they ever need to be updated, or are they specific to this extension and ASF?

Reply:
I meant to delete this out of the package.json. I think it maybe got added when I had been testing the debugger locally but also have the extension installed from the marketplace as I am not sure 100% where it came from. Would be deleting it be fine?

Reply:
I don't really know what this __metadata is used for, so hard to say if it should be deleted. But we haven't needed it in the past so I assume it's safe to remove.